dev/PullToRefresh/RefreshVisualizer/RefreshVisualizer.cpp

```diff
@@ -298,7 +298,13 @@ void RefreshVisualizer::UpdateContent()
             contentVisual.Opacity(1.0f);
             if (m_root)
             {
-                float translationRatio = (1.0f - (float)(m_refreshInfoProvider.get().ExecutionRatio())) * PARALLAX_POSITION_RATIO;
+                float translationRatio = [this]() {
+                    if (auto&& refreshInfoProvider = m_refreshInfoProvider.get())
+                    {
+                        return (1.0f - (float)(refreshInfoProvider.ExecutionRatio())) * PARALLAX_POSITION_RATIO;
+                    }
+                    return 0.0f;
```

Comment:
A default translation ratio of 0.0 is probably wrong. It means that a refresh will be triggered when we have pulled 0% of the visualizers height, I'm not sure what the symptoms of this would be but I think a 1.0 default value (triggers a refresh when 100% of the visualizers length has been pulled) would be safer.